greenbone/feed/sync/main.py

```diff
@@ -69,13 +70,45 @@ def filter_syncs(
     )
 
 
+def do_selftest(error_console: Console) -> int:
+    """
+    Check for sha256sum and rsync commands.
+    """
+    try:
+        subprocess.run(
+            ["sha256sum", "--help"],
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL,
+            check=True,
+        )
+    except PermissionError:
```

Comment:
Shouldn't this be a [subprocess.CalledProcessError](https://docs.python.org/3/library/subprocess.html#subprocess.CalledProcessError)
```suggestion
    except subprocess.CalledProcessError:
```

Reply:
on it!